src/index.js

```diff
@@ -2,18 +2,23 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
+
+
 import { Provider } from "react-redux";
-import { createStore } from "redux";
+import { createStore, applyMiddleware} from "redux";
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
 import rootReducer from "./reducers";
 // needed dependancies
 // applyMiddleware from redux
 // thunk from redux-thunk
 // logger from redux-logger
 // rootReducer from ./reducers
 
+const middleware = applyMiddleware(logger, thunk)
```

Comment:
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const store = createStore(
  rootReducer,
  composeEnhancers(applyMiddleware(thunk, logger))
);

 If you wanted to use redux tools.